### PR TITLE
[IMP] business_requirement: Improve module business_requirement based…

### PR DESCRIPTION
**Pull request by martintorresvm**
… on BR000168 - Task 12797
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/xie8899/project/pull/38%23issuecomment-201596170%22%2C%20%22https%3A//github.com/xie8899/project/pull/38%23discussion_r57542092%22%2C%20%22https%3A//github.com/xie8899/project/pull/38%23discussion_r57553926%22%2C%20%22https%3A//github.com/xie8899/project/pull/38%23discussion_r57671136%22%2C%20%22https%3A//github.com/xie8899/project/pull/38%23discussion_r57674083%22%2C%20%22https%3A//github.com/xie8899/project/pull/38%23discussion_r57677028%22%2C%20%22https%3A//github.com/xie8899/project/pull/38%23issuecomment-202741145%22%2C%20%22https%3A//github.com/xie8899/project/pull/38%23issuecomment-202750019%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/38%23issuecomment-201596170%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-03-25T23%3A52%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%2C%20%7B%22body%22%3A%20%22Is%20this%20one%20OK%3F%22%2C%20%22created_at%22%3A%20%222016-03-29T06%3A38%3A35Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGT%3C%22%2C%20%22created_at%22%3A%20%222016-03-29T07%3A13%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20b0920e46ab3b61b1580385beea3dab36095f593a%20business_requirement/models/business.py%2041%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/38%23discussion_r57542092%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40victormartinelicocorp%20%20When%20we%20use%20the%20wizard%20to%20create%20the%20tasks%2C%20this%20field%20should%20be%20automatically%20assigned.%22%2C%20%22created_at%22%3A%20%222016-03-28T03%3A26%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40duanyp1991%20actually%20works%20that%20way%20I%20pass%20the%20context%20and%20it%27s%20filled.%20Maybe%20should%20I%20get%20context%20here%20and%20not%20in%20the%20view%3F%20%5Cr%5Cncc%20%40duanyp1991%20%22%2C%20%22created_at%22%3A%20%222016-03-28T08%3A22%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16461806%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/victormartinelicocorp%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20meant%20in%20the%20wizard%20%5C%22generate%20projects%5C%22%22%2C%20%22created_at%22%3A%20%222016-03-29T04%3A36%3A15Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40duanyp1991%20Then%20I%27ll%20create%20a%20new%20version%20for%20the%20related%20module%20business_requirement_project%5Cr%5Cn%5Cr%5Cncc%20%40elicoidal%20%22%2C%20%22created_at%22%3A%20%222016-03-29T05%3A40%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16461806%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/victormartinelicocorp%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40duanyp1991%20%5Cr%5Cnhttps%3A//github.com/xie8899/project/pull/42%5Cr%5Cn%5Cr%5Cncc%20%40elicoidal%20%40duanyp1991%20%22%2C%20%22created_at%22%3A%20%222016-03-29T06%3A28%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16461806%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/victormartinelicocorp%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20business_requirement/models/business.py%3AL468-482%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/38#issuecomment-201596170'>General Comment</a></b>
- <a href='https://github.com/elicoidal'></a> LGTM
- <a href='https://github.com/elicoidal'></a> Is this one OK?
- <a href='https://github.com/duanyp1991'></a> LGT<
- [ ] <a href='#crh-comment-Pull b0920e46ab3b61b1580385beea3dab36095f593a business_requirement/models/business.py 41'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/38#discussion_r57542092'>File: business_requirement/models/business.py:L468-482</a></b>
- <a href='https://github.com/duanyp1991'></a> @victormartinelicocorp  When we use the wizard to create the tasks, this field should be automatically assigned.
- <a href='https://github.com/victormartinelicocorp'></a> @duanyp1991 actually works that way I pass the context and it's filled. Maybe should I get context here and not in the view?
  cc @duanyp1991
- <a href='https://github.com/duanyp1991'></a> I meant in the wizard "generate projects"
- <a href='https://github.com/victormartinelicocorp'></a> @duanyp1991 Then I'll create a new version for the related module business_requirement_project
  cc @elicoidal
- <a href='https://github.com/victormartinelicocorp'></a> @duanyp1991
  https://github.com/xie8899/project/pull/42
  cc @elicoidal @duanyp1991

<a href='https://www.codereviewhub.com/xie8899/project/pull/38?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/xie8899/project/pull/38?approve=1'></a>&nbsp;<a href='https://github.com/xie8899/project/pull/38'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by xie8899** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/xie8899/project/pull/38'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by elicoidal** [comment]
LGTM

File: business_requirement/models/business.py [hunk 1]
```diff
@@ -453,3 +468,14 @@ class Project(models.Model):
     @api.depends('br_ids')
     def _br_count(self):
         self.br_count = len(self.br_ids)
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    """ Link the task and the business requirement """
+    business_requirement_id = fields.Many2one(
```
Comment (duanyp1991) [inline business_requirement/models/business.py:477]: @victormartinelicocorp  When we use the wizard to create the tasks, this field should be automatically assigned.

Comment (martintorresvm) [inline business_requirement/models/business.py:477]: @duanyp1991 actually works that way I pass the context and it's filled. Maybe should I get context here and not in the view? 
cc @duanyp1991

Comment (duanyp1991) [inline business_requirement/models/business.py:477]: I meant in the wizard "generate projects"

Comment (martintorresvm) [inline business_requirement/models/business.py:477]: @duanyp1991 Then I'll create a new version for the related module business_requirement_project

cc @elicoidal

Comment (martintorresvm) [inline business_requirement/models/business.py:477]: @duanyp1991 
https://github.com/xie8899/project/pull/42

cc @elicoidal @duanyp1991

**Comment by elicoidal** [comment]
Is this one OK?

**Comment by duanyp1991** [comment]
LGTM

**Pull request merged**